### PR TITLE
Whole library is now using namespaces

### PR DESCRIPTION
**Pull request by MarioBlazek**
This PR adds namespaces to library. 

Due to restrictions of PHP some clases needed renaming:
* src/Builder/Namespace.php -> src/Builder/NamespaceMatch.php                                                                                                         
* src/Invocation/Object.php -> src/Invocation/ObjectInvocation.php
* src/Invocation/Static.php -> src/Invocation/StaticInvocation.php
* src/Stub/Return.php -> src/Stub/ReturnStub.php

TODO
- [x] update tests

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -35,9 +35,9 @@
         "ext-soap": "*"
     },
     "autoload": {
-        "classmap": [
-            "src/"
-        ]
+        "psr-4": {
```
Comment (sebastianbergmann) [inline composer.json]: Please revert this change. I do not use PSR-4 in my software projects.

**Comment by MarioBlazek** [comment]
@sebastianbergmann what is your opinion about this PR ? I need your advice about tests, what would be the best approach ? Thanks.

**Comment by sebastianbergmann** [comment]
I've been meaning to namespace the classes of this component since before PHPUnit 6 came out but did not have time to do so. So, yes, I want this.

**Comment by sebastianbergmann** [comment]
What advice do you need for adapting the tests?

**Comment by MarioBlazek** [comment]
@sebastianbergmann I've adapted tests, why do you think ?

**Comment by sebastianbergmann** [comment]
Thank you for your work so far on this, @MarioBlazek. I'll take a closer look at it next week.

**Comment by MarioBlazek** [comment]
Great, thanks @sebastianbergmann.

**Comment by MarioBlazek** [comment]
@sebastianbergmann did you manage to check this PR ?

**Comment by sebastianbergmann** [comment]
It's almost at the top of my stack of things to look at. I'm sorry that I have not gotten to it yet.

**Comment by sebastianbergmann** [comment]
Merged manually, thanks.

**Comment by MarioBlazek** [comment]
Thank you for merge @sebastianbergmann .
